cfme/networks/provider/nuage.py

```diff
@@ -69,9 +83,14 @@ def mgmt(self):
 
     @classmethod
     def from_config(cls, prov_config, prov_key, appliance=None):
```

Comment:
Would you add a docstring here.   Every little bit helps.

Reply:
Good point, thanks, done.

Reply:
sweet